lib/db/memory.js

```diff
@@ -135,12 +136,14 @@ MemoryStore.prototype = {
     delete this.clients[unbuf(id)];
     return P.resolve();
   },
-  generateCode: function generateCode(clientId, userId, email, scope) {
+  generateCode: function generateCode(clientId, userId, email, scope,
+                                      lastAuthAt) {
```

Comment:
aye; I considered passing in a "claims" object with both email and lastAuthAt as properties, but this seemed like a layering violation